### PR TITLE
Add functional test for unsupported provisioners

### PR DESCRIPTION
**Pull request by akalenyu**
Signed-off-by: Alex Kalenyuk <akalenyu@redhat.com>

**What this PR does / why we need it**:

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: tests/operator_test.go [hunk 1]
```diff
@@ -1025,6 +1026,28 @@ var _ = Describe("ALL Operator tests", func() {
 				}
 			})
 
+			It("StorageProfile incomplete metric expected value remains unchanged for provisioner known to not work", func() {
```
Comment (duyanyan) [inline tests/operator_test.go:1029]: [test_id:9659]

Comment (akalenyu) [reply tests/operator_test.go]: done

**Pull request merged**